public/projects/installfest.md

```diff
@@ -62,72 +60,231 @@ and sign up for accounts on:
 6. **Double-click** the shortcut to open the terminal
 7. Press <kbd>Win</kbd>-<kbd>Alt</kbd>-<kbd>P</kbd> (or select "Settings..." from the window's menu) and make sure the "startup task" is set to `{cmd::Cmder as Admin}`
 
-## VS Code Terminal
-1. Open VS Code
-2. Press <kbd>Crtl</kbd>-<kbd>Shift</kbd>-<kbd>P</kbd>, and type in `Default Shell`
-3. Choose  `cmd` or `Git Bash`
+### MongoDB and Mongo Compass
+
+#### MongoDB Service
+
+1. Visit the MongoDB installation page at <https://docs.mongodb.com/manual/tutorial/install-mongodb-on-windows/>
+2. Download the MongoDB Community Server at: <https://www.mongodb.com/try/download/community>
+3. Run the installer from your Downloads folder
+4. Choose the "Complete" setup type from the installation wizard
+5. On the "Service Configuration" pane, select "Run Service as Network Service User"
+
+#### MongoDB Compass
+
+1. Visit the MongoDB Compass installation instructions <https://docs.mongodb.com/compass/master/install/>
+2. Download MongoDB Compass from <https://www.mongodb.com/try/download/compass>
+3. Run the installer from your Downloads folder
+4. Accept the default values for installation location and options
```

Comment:
MongoDB Compass comes bundled with the complete version of the MongoDB community server so they shouldn't need to install it separately.